[1,"\u001B[0m\u001B[1m\u001B[31merror\u001B[0m: Uncaught Error: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--11) property name \u001B[34mlanguage\u001B[39m is invalid\n5: \u001B[34m  title: \"מבוא למדעי הרוח הדיגיטליים / שי גורדין\"\u001B[39m\n6: \u001B[34m  language: he\u001B[39m\n    ~~~~~~~~~\n7: \u001B[34m  dir: rtl\u001B[39m\n\u001B[34mℹ\u001B[39m The error happened in location \u001B[34mwebsite\u001B[39m:\u001B[34mlanguage\u001B[39m.\n\n\nIn file _quarto.yml\n(line 7, columns 3--6) property name \u001B[34mdir\u001B[39m is invalid\n6: \u001B[34m  language: he\u001B[39m\n7: \u001B[34m  dir: rtl\u001B[39m\n    ~~~~\n8: \u001B[34m  navbar:\u001B[39m\n\u001B[34mℹ\u001B[39m The error happened in location \u001B[34mwebsite\u001B[39m:\u001B[34mdir\u001B[39m.\n\n        throw new ValidationError2(errorMessage, yamlValidationErrors);\n\u001B[0m\u001B[31m              ^\u001B[0m\n    at \u001B[0m\u001B[1m\u001B[3mreadAndValidateYamlFromFile\u001B[0m (\u001B[0m\u001B[36mfile:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m42120\u001B[0m:\u001B[0m\u001B[33m15\u001B[0m)\n    at async \u001B[0m\u001B[36mfile:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m82871\u001B[0m:\u001B[0m\u001B[33m26\u001B[0m\n    at async \u001B[0m\u001B[1m\u001B[3mprojectContext\u001B[0m (\u001B[0m\u001B[36mfile:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m82723\u001B[0m:\u001B[0m\u001B[33m26\u001B[0m)\n    at async \u001B[0m\u001B[1m\u001B[3mrender\u001B[0m (\u001B[0m\u001B[36mfile:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m90654\u001B[0m:\u001B[0m\u001B[33m21\u001B[0m)\n"]
